ecommerce/templates/oscar/basket/basket.html

```diff
@@ -45,12 +35,19 @@
                 {% if basket.is_empty %}
                     {% block emptybasket %}
                         <div class="depth depth-2 message-error-content">
-                            <h3>{% trans "Your basket is empty" as tmsg %}{{ tmsg | force_escape }}</h3>
-                            {% blocktrans asvar tmsg %}
-                                If you attempted to make a purchase, you have not been charged. Return to your {link_start}{link_middle}{homepage_url}dashboard{link_end} to try
-                                again, or {link_start}{homepage_url}{link_middle}contact {platform_name} Support{link_end}.
+                            <h3>{% trans "Your basket is empty" %}</h3>
+                            {% captureas dashboard_link_start %}
+                                <a class="basket-homepage-link" href="{{ homepage_url }}">
+                            {% endcaptureas %}
+
+                            {% captureas support_link_start %}
+                                <a class="basket-support-link" href="{{ support_url }}">
+                            {% endcaptureas %}
+
+                            {% blocktrans with link_end="</a>" %}
+                                If you attempted to make a purchase, you have not been charged. Return to your {{ dashboard_link_start }}dashboard{{ link_end }} to try
```

Comment:
@tatianatorog can you please check if this is correct?
{{ dashboard_link_start }}dashboard{{ link_end }}

Reply:
@magajh , estoy haciendo el despliegue en stage ya que mis cambios ya no están , debido a un deploy que hizo Jhony.

Reply:
@tatianatorog ya lo probé, gracias!